src/debug.c

```diff
@@ -1024,6 +1024,10 @@ void bugReportStart(void) {
 
 #ifdef HAVE_BACKTRACE
 static void *getMcontextEip(ucontext_t *uc) {
+#define NOT_SUPPORTED() do {\
+    UNUSED(uc);\
+    return NULL;\
+} while(0)
```

Comment:
why not just always do `UNUSED(uc)`, and have a single `return NULL` at the end?

Reply:
If I add a `return NULL;` at the end then when the arch is known it'll compile to two returns:
```c
return (void*)uc->stuff;
return NULL;
```
Which will generate more warnings.

Reply:
ohh.. unreachable code...
well, ok. let's keep your version..